_posts/2023-09-13-make-meaningful-contributions-at-osd.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Vlog: Join us at Open Source Day on September 22nd, 2023 and Level Up Your Skills'
+date: 2023-09-13
+tags: ghc ghc23 osd opensourceday
+synopsis: Check out this short video to learn how you can join the WildFly Elytron project and make meaningful contributions to the open source community!
+author: szaldana
```

Comment:
should be `soniazaldana` 

https://github.com/wildfly/wildfly.org/blob/1a8251669d1458fa9e8215e047c6948d7f70016f/_data/authors.yaml#L123C1-L123C13